src/components/venues/Jazzbar/components/MessageList.tsx

```diff
@@ -16,29 +16,19 @@ const MessageList: React.FC<MessageListProps> = ({ messages }) => {
   }));
   const [selectedUserProfile, setSelectedUserProfile] = useState<User>();
 
-  const profileImageSize = 40;
   return (
     <>
       <div className="reaction-list small">
         {messages.map((message) => (
-          <div className="message" key={`${message.from}-${message.ts_utc}`}>
-            <img
-              onClick={() =>
-                setSelectedUserProfile({
-                  ...usersById[message.from],
-                  id: message.from,
-                })
-              }
-              key={`${message.from}-messaging-the-band`}
-              className="profile-icon"
-              src={usersById[message.from].pictureUrl || DEFAULT_PROFILE_IMAGE}
-              title={usersById[message.from].partyName}
-              alt={`${usersById[message.from].partyName} profile`}
-              width={profileImageSize}
-              height={profileImageSize}
-            />
-            <div className="message-bubble">{message.text}</div>
-          </div>
+          <>
+            {message.from in usersById && (
+              <Message
+                sender={usersById[message.from]}
+                message={message}
+                onClick={() => setSelectedUserProfile(usersById[message.from])}
```

Comment:
Not sure you can change this. The `id` is (was) used in the modal.

Reply:
sender prop gives user